### DISCUSSION TITLE
ABILITY TO DELETE HISTORY AT CLICK OF A BUTTON!!!!

### DISCUSSION THREAD

**Discussion by ghost** [vrcx-team/VRCX#623] [Ideas]
i think it would be pretty cool to make it possible to clear feed, game log.. overall history of join count, who you have unfriended, requested friend request, received invites from, seeing who changed the video in your world etc too .. instead of manually doing it (which is only possible for the friend log tab) or deleting your pc account and making a new one.

i would like to just click a button instead of deleting my entire account on my pc lol i think it would be helpful as i regularly do this and i have to keep installing and uninstalling games just to tame my historyless ocd. think many people could benefit from this too not just me <3

think it would be real cool, i think this could be possible.

**Comment by Natsumi-sama** [comment]
The database file is stored here `%AppData%\VRCX\VRCX.sqlite3` you can delete the file or manually delete tables using https://sqlitebrowser.org/

**Comment by ghost** [comment]
ok, but thats complex stuff. it would be easier built in imo

**Reply by Natsumi-sama** [reply]
I think you're the only person who would want this feature and deleting a file isn't complex.

**Reply by Myrkie** [reply]
I have to agree with natsumi, basic file operations are expected from anyone who uses a computer.

**Comment by ghost** [comment]
i ws talking about using this sqlitebrowser link thing

**Reply by LucHeart** [reply]
Well that was only half of the answer, if thats to complex, just delete the file as stated.
Dont see a demand for this feature.
